README.md

```diff
@@ -60,6 +60,18 @@ Please find the spark-expectations flow and feature diagrams below
 <img src=https://github.com/Nike-Inc/spark-expectations/blob/main/docs/se_diagrams/features.png?raw=true width=1000></p>
 
 
+# Observability Enhancement: Additional Column for Data Quality Metrics
```

Comment:
Please move this documentation to the documentation under docs folder, so that it refelects in SE documentation

Reply:
keeping the readme file as it is as reverting back the column name changes.
Will update the documentation with the further releases.